src/steps/finish.ts

```diff
@@ -18,7 +19,7 @@ async function createFinish(
 ) {
   const {
     log,
-    coreArgs: { description, logsURL },
+    coreArgs: { description, logsURL, environment },
   } = context;
```

Comment:
Thanks for the contribution, but this change doesn't build. This environment is from context, whereas it seems you are trying to parameterize it as an action input? 

Either way, both don't align with the `createDeploymentStatus` which only allows a set of values `"production" | "staging" | "qa"` so this doesn't build

Reply:
I've reverted the changes for now